src/aggregator/aggregator/generic_elem.go

```diff
@@ -468,10 +468,13 @@ func (e *GenericElem) processValueWithAggregationLock(
 		discardNaNValues = e.opts.DiscardNaNAggregatedValues()
 	)
 	for aggTypeIdx, aggType := range e.aggTypes {
+		toFlush := make([]transformation.Datapoint, 0)
```

Comment:
Can we place this slice somewhere on the element or in the locked aggregation so we can reuse it each flush? Ideally we don't need to allocate this every flush per every single series.

Alternatively, can we just create two local variables? `toFlushDefault` and `toFlushSecondary`